vm-setup/roles/v1aX_integration_test/files/fetch_manifests.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Fetch manifests
+declare -a resource=("bmh" \
+"cluster" \
+"deployment" \
+"machine" \
+"m3cluster" \
+"machinedeployment" \
+"machinehealthchecks" \
+"machinesets" \
+"machinepools" \
+"m3machine" \
+"metal3machinetemplate" \
+"kcp" \
+"kubeadmconfig" \
+"kubeadmconfigtemplates" \
+"kubeadmcontrolplane" \
+"replicaset")
+
+if [[ "${CAPI_VERSION}" == "v1alpha3" ]]; then 
```

Comment:
that looks incorrect, no ? those are v1a4 resources

Reply:
True. Thanks